src/index.js

```diff
@@ -1,10 +1,40 @@
 const { Telegraf } = require("telegraf");
 require("dotenv").config();
+const Groq = require("groq-sdk");
 
 const bot = new Telegraf(process.env.BOT_TOKEN);
+const groq = new Groq();
 
 bot.start((ctx) => ctx.reply("Welcome to your AI Assistant!"));
 
+// Handle messages and respond using Groq SDK
+bot.on("text", async (ctx) => {
+  const userMessage = ctx.message.text;
+
+  try {
+    // Use Groq's chat completion to generate a response
+    const chatCompletion = await groq.chat.completions.create({
```

Comment:
**praise**: The use of Groq's chat completion API is well-implemented. This is a crucial part of the task and is handled correctly.